core/templates/dev/head/pages/collection_editor/CollectionEditorStateService.ts

```diff
@@ -18,10 +18,10 @@
  * retrieving the collection, saving it, and listening for changes.
  */
 
-oppia.constant('EVENT_COLLECTION_INITIALIZED', 'collectionInitialized');
-oppia.constant('EVENT_COLLECTION_REINITIALIZED', 'collectionReinitialized');
+angular.module('collectionEditorModule').constant('EVENT_COLLECTION_INITIALIZED', 'collectionInitialized');
+angular.module('collectionEditorModule').constant('EVENT_COLLECTION_REINITIALIZED', 'collectionReinitialized');
```

Comment:
Break this into multiple lines.

Reply:
Hi Yash, I will fix all the lint errors after the PR passes e2e tests as discussed in admin page PR.
Thanks!